tests/run_tests_headless.sh

```diff
@@ -19,52 +19,13 @@ cd "${HERE}"
 EXECUTABLE="../endless-sky"
 RESOURCES="../"
 
-
-Xvfb :99 -screen 0 1280x1024x24 &
-XSERVER_PID=$!
-if ! ps -p ${XSERVER_PID} > /dev/null 2>&1; then
+if ! command -v xvfb-run &> /dev/null; then
```

Comment:
`&>` is a bashism. There's nothing in this script that truly requires bash, so I would recommend switching this back to `> /dev/null 2&>1` (which works on all Bourne shells, including bash), and changing the shebang (line 1) to the more widely available `#!/bin/sh`. The conditionals on line 9 should also be changed from `[[ X == 'Y' ]]` to `[ "X" = 'Y' ]`, for greater compatibility.

Reply:
If I change the conditionals I get `./tests/run_tests_headless.sh: line 9: [: =: unary operator expected`.

Reply:
Scratch that I can't read.